src/GraphGen/GraphVisNode.cs

```diff
@@ -25,13 +28,37 @@ public GraphVisNode(ProjectGraphNode node)
         internal string Create()
         {
             
-            var globalPropertiesString = string.Join("\n", _node.ProjectInstance.GlobalProperties.OrderBy(kvp => kvp.Key).Where(kvp => kvp.Key != "IsGraphBuild").Select(kvp => $"{kvp.Key}={kvp.Value}"));
+            var globalPropertiesString = string.Join("\n", Node.ProjectInstance.GlobalProperties.OrderBy(kvp => kvp.Key).Where(kvp => kvp.Key != "IsGraphBuild").Select(kvp => $"{kvp.Key}={kvp.Value}"));
             if (globalPropertiesString.StartsWith("TargetFramework="))
             {
                 globalPropertiesString = globalPropertiesString.Substring("TargetFramework=".Length);
             }
 
-            return $"  {Name} [label=\"{_label}\n{globalPropertiesString}\", shape=box];"; //, color=\"0.650 0.200 1.000\"];";
+            if (globalPropertiesString != "")
+            {
+                globalPropertiesString = "\n" + globalPropertiesString;
+            }
+
+            var buildTargetsString = "";
+            if (Targets != null)
+            {
+                buildTargetsString = "\nTargets={" + string.Join(",", Targets.Distinct()) +"}";
```

Comment:
Can remove .Distinct(), they should already be distinct, probably a bug if they aren't.